README.md

```diff
@@ -0,0 +1,39 @@
+# Shardeum Network Status
+
+This repo is developed to check for the Status of the Shardeum Network services.
+
+This is to ensure whether the services are running are turned off.
+
+### Getting Started.
+To get started, ensure you have `Node.js` installed preferably v18+ 
+
+### Installation
+1. Clone the Network Status Repository.
+   
+> First clone the shardeum network Status repository to your local machine.
+```
+git clone https://github.com/shardeum/network-status.git
+
+```
+2. Install the required dependencies for the frontend and backend:
+   > Fo the backend,
+   ```
+     cd backend
+      npm i
+    ```
+    This will install all the dependencies required for the backend
+
```

Comment:
Installing the deps for both the frontend and backend is a good approach but it might be better to finish the set up for one before moving onto the next. For instance, after installing the dependencies for the backend project, start the development server before moving onto the frontend. Here's a suggestion to include:

```suggestion
To start the local development server for the backend project, run the following command:
npm start
This will start the local Node server on port `3002` (or your locally specified port). 
> Your Prometheus server should be running on port 9090  [how to set up Prometheus](https://github.com/shardeum/network-status/tree/main/backend#run-prometheus)

```